packages/wallets/src/components/WalletGraph.tsx

```diff
@@ -173,63 +132,37 @@ export default function WalletGraph(props: WalletGraphProps) {
     walletId,
   });
 
-  const [ref, containerSize] = useMeasure();
-
+  const isCAT = [WalletType.CAT, WalletType.CRCAT].includes(walletType);
   const isLoading = isWalletTransactionsLoading || isWalletBalanceLoading || !transactions;
-  if (isLoading || !walletBalance) {
-    return null;
-  }
 
-  const confirmedTransactions = transactions.filter((transaction) => transaction.confirmed);
-  if (!confirmedTransactions.length) {
-    return null;
-  }
+  const confirmedTransactions = transactions ? transactions.filter((transaction) => transaction.confirmed) : [];
 
-  const balance = walletBalance.confirmedWalletBalance;
+  const balance = walletBalance?.confirmedWalletBalance || 0;
 
-  const data = prepareGraphPoints(balance, confirmedTransactions, walletType, {
-    interval: 60 * 60,
-    count: 24,
-    offset: 0,
-  });
+  const data = prepareGraphPoints(balance, confirmedTransactions);
+
+  const xValueFormatter = useCallback((value: number) => moment(value * 1000).format('LLL'), []);
 
-  const min = data.length ? Math.min(...data.map((item) => item.y)) : 0;
-  const max = Math.max(min, ...data.map((item) => item.y));
+  const yValueFormatter = useCallback(
+    (value) => {
```

Comment:
My VSCode complains a missing type.
According to the `yValueFormatter` definition by `LineChart`, this should be `(value: number | BigNumber | null) => {`